### ISSUE TITLE
Add Softmax layer to prediction model 

### ISSUE THREAD

**Issue by NotJoeMartinez** [NotJoeMartinez/APDL_Quality_Control#19]
This will fix the issue with the predictions confidence being all wacky. You implemented this in the [API](https://github.com/NotJoeMartinez/APDL_QC_API/blob/main/scripts/predict.py). Add it to `test_model.py`

**Comment by NotJoeMartinez** [comment]
Fixed in cd5807c

```python 
probability_model = tf.keras.Sequential([model, tf.keras.layers.Softmax()])
```
